Stylesheets/PageStyles/home.css

```diff
@@ -0,0 +1,135 @@
+.main {
+   
+}
+
+.product-content {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+}
+
+.product-gryffindor {
+    background-color: #800000;
+    width: 100%;
+    height: 100vh;
+}
+
+.product-slytherin {
+    background-color: #294936;
+    width: 100%;
+    height: 100vh;
+}
+
+.product-ravenclaw {
+    background-color: #015783;
+    width: 100%;
+    height: 100vh;
+}
+
+.product-hufflepuff {
+    background-color: #FFD63B;
+    width: 100%;
+    height: 100vh;
+}
+
+.product {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column
+}
+
+.product-btn {
+    color: var(--st-white);
+    padding: 1rem;
+    border: 2px solid var(--st-white);
+    border-radius: 0.5rem;
+    margin: 1rem;
+}
+
+.new-product {
+    display: flex;
+    padding: 0.5rem;
+    align-items: center;
+    margin: 0.5rem;
+    border-radius: 0.5rem;
+    box-shadow: 2px 2px 3px hsla(200, 100%, 15%, 0.1); 
+    background-color: hsla(0, 0%, 100%, 0.3);
+    backdrop-filter: blur(4px);
+    cursor: pointer;
+}
+
+.small-card-img {
+    width: 6rem;
+    height: 6rem;
+}
+
+.product-btn:hover {
+    color: var(--st-black);
+    border: 2px solid var(--st-black);
+}
+
+.new-product:hover {
+    background-color: hsla(0, 0%, 100%, 0.6);
+}
+
+@media only screen and (max-width: 920px) {
+    .product-content {
+        grid-template-columns: repeat(2, 1fr);
+    }
+
+    .product-gryffindor {
+        background-color: #800000;
+        width: 100%;
+        height: 50vh;
+    }
+    
+    .product-slytherin {
+        background-color: #294936;
+        width: 100%;
+        height: 50vh;
+    }
+    
+    .product-ravenclaw {
+        background-color: #015783;
+        width: 100%;
+        height: 50vh;
+    }
+    
+    .product-hufflepuff {
+        background-color: #FFD63B;
+        width: 100%;
+        height: 50vh;
+    }
+}
+
+@media only screen and (max-width: 520px) {
```

Comment:
any reason for having different sizes of media queries

Reply:
Yes, because of my home page design. It does not look good when the screen size decreases, the contents are getting overlapped.